packages/tools/pluggable-widgets-tools/configs/rollup-plugin-widget-typing.js

```diff
@@ -6,13 +6,14 @@ const { transformPackage } = require("../dist/typings-generator");
 
 export function widgetTyping({ sourceDir }) {
     let firstRun = true;
+    let filePaths = [];
```

Comment:
What is your reason for going with an array here? We only care about the props typing file, right?

Reply:
In theory we support more than 1 file, for multiple widgets in this case.

Reply:
Aha, does PIW-tools support multiple widgets in the package definition?

Reply:
As I understand from Diego, PIW-tools does have some support built-in for multiple widgets defined in a `package.xml, so let's keep the array. @wesleysieses Could you rename the variable to something like: `propsTypingFilePaths`?

Reply:
Updated ✅